src/client/actions/user/index.ts

```diff
@@ -288,6 +315,45 @@ const updateLongUrl = (shortUrl: string, longUrl: string) => (
   })
 }
 
+// API call to update description and contact email
+const updateUrlInformation = (shortUrl: string) => (
+  dispatch: ThunkDispatch<
+    GoGovReduxState,
+    void,
+    SetErrorMessageAction | SetSuccessMessageAction
+  >,
+  getState: GetReduxState,
+) => {
+  const { user } = getState()
+  const url = user.urls.filter(
+    (urlToCheck) => urlToCheck.shortUrl === shortUrl,
+  )[0]
```

Comment:
nitpick - prefer array destructuring over indexing
```suggestion
  const [ url ] = user.urls.filter(
    (urlToCheck) => urlToCheck.shortUrl === shortUrl,
  )
```